### PR TITLE
add tmp comments

### PR DESCRIPTION
**Pull request by BeataKula**

### CODE REVIEW

**Review by zgr92 — APPROVED** [review]

File: src/components/Button.tsx [hunk 1]
```diff
@@ -30,3 +30,4 @@ const Button: ComponentType<ButtonType> = ({ id, name, about, onClick }) => {
 };
```
Comment (zgr92) [inline src/components/Button.tsx:30]: ComponentType -> FunctionComponent, the same in other places with ComponentType, it should be replaced

File: src/App.tsx [hunk 2]
```diff
@@ -6,6 +6,8 @@ import Message, { categoryType } from "./components/Message";
 
 import { Wrapper, Title } from "./components/AppStyles";
 
+/* Tylko dla PR - tmp */
+
 import "./App.css";
```
Comment (zgr92) [inline src/App.tsx:12]: `<Title>
       <h1>Blog Beaty</h1>
</Title>`

h1 here is redundant

File: src/components/Button.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (BeataKula) [reply src/components/Button.tsx:30]: hmm it was for the future ...but I realised, that don't know, how get hover effect for this button by styled-components...

File: src/components/LinkComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    className: string;
+    children: string;
+};
+
+const LinkComponent: FunctionComponent<LinkProps> = ({
+    to,
+    className,
+    children,
+}) => (
+    <Link to={to} className={className}>
```
Comment (zgr92) [inline src/components/LinkComponent.tsx:16]: No class name if we want to introduce styled components. No need

File: src/pages/NoPage.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React, { FunctionComponent } from "react";
+
+type ContactProps = {
+    isActive: boolean;
+};
+
+const NoPage: FunctionComponent<ContactProps> = ({ isActive }) => {
+    const isActiveFlag = isActive ? " active " : "";
```
Comment (zgr92) [inline src/pages/NoPage.tsx:8]: No function for active

File: src/pages/PostsListPage.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    render() {
+        return (
+            <div className="ui relaxed divided list">{this.renderList()}</div>
+        );
+    }
+}
+
+const mapStateToProps = (state) => {
```
Comment (zgr92) [inline src/pages/PostsListPage.js:30]: Does it do anything?

File: src/AppTypes.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
+export interface IArticle {
```
Comment (zgr92) [inline src/AppTypes.tsx:1]: It can be a .ts file since you don't use the syntax like <Component /> here

File: src/AppTypes.tsx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    color: String;
+    size: "mini" | "tiny" | "small" | "large" | "big" | "huge" | "massive";
+};
+
+export interface PostsListPageType {
+    fetchPosts: FetchPostType;
+    posts: IPost[];
+}
+
+export interface IPost {
```
Comment (zgr92) [inline src/AppTypes.tsx:36]: No need to mark interface with "I" in "IPost" in Typescript. This is used in some languages, but not here

File: src/pages/PostsListPage.js (hunk 5 above)
Comment (BeataKula) [reply src/pages/PostsListPage.js]: Yes it returns { posts: state.posts };

File: src/AppTypes.tsx (hunk 7 above)
Comment (BeataKula) [reply src/AppTypes.tsx:36]: Ok, it Was from: 
obraz

Comment (BeataKula) [reply src/AppTypes.tsx:36]: The name **Post** was used in the **Post** component, so it seems to be the best name in this case

**Pull request merged**